6-fancy-hello-world/solution.js

```diff
@@ -0,0 +1,13 @@
+const input = document.querySelector(".controls__input");
+const h1 = document.querySelector(".header__text");
+
+input.addEventListener("change", updateHeaderText);
+
+function updateHeaderText(event) {
+  const { value } = event.target;
+  h1.textContent = value;
+
+  if (h1.textContent === "" || h1.textContent === undefined) {
```

Comment:
```suggestion
  if (h1.textContent === "" ) {
```
Should be sufficient here, since it is never `undefined`